synthesizer/src/vm/helpers/cost.rs

```diff
@@ -55,7 +55,7 @@ pub fn deployment_cost<N: Network>(deployment: &Deployment<N>) -> Result<(u64, (
 pub fn execution_cost<N: Network, C: ConsensusStorage<N>>(
     vm: &VM<N, C>,
     execution: &Execution<N>,
-) -> Result<(u64, (u64, u64))> {
+) -> Result<(u64, (u64, u64, u64))> {
```

Comment:
I wonder if we shouldn't introduce some new `struct` to minimize the chance of mixing these values up?

Reply:
Or we don't return anything anymore: 6d0d4c986
(When creating a type for it the compiler complained it was never read)

Reply:
From a UI perspective it's actually prudent to return the cost breakdown so that people can plan their cost. The SDK uses all the values of this function for instances. It'd be wise to have a struct from the perspective of communication.